src/DynamoMLDataPipeline/DynamoMLDataPipeline.cs

```diff
@@ -147,12 +167,8 @@ static public void EndFullFillment(RestClient client, string collectionId, strin
 
```

Comment:
This comment is for lines 154-165: I know that it is outside the scope of the changes, but would it make sense to completely remove the while block? This was serving for testing, to confirm if the fulfillment was successfully completed. Since there is no follow up action dependent on the status of fulfillment why do we keep it? It would make sense to keep it if we want to keep track of successful versus not fulfillments, for sending an Analytics log for example. 
And even in that case, if something goes wrong on the Forge side and the fulfillment never succeeds, we will get stuck in the while loop.

Reply:
Yes removed the while loop.